src/ecpair.js

```diff
@@ -68,7 +68,7 @@ ECPair.fromWIF = function (string, network) {
   if (types.Array(network)) {
     network = network.filter(function (x) {
       return version === x.wif
-    }).pop()  // We should not use pop since it depends on the order of the networks for the same wif
```

Comment:
if there is more than one match, what is the alternative to using the last element? any other pick is just as arbitrary